package.json

```diff
@@ -11,7 +11,8 @@
     "webpack-analyze": "ng build --sourceMap=true --namedChunks=true --stats-json && ./node_modules/.bin/webpack-bundle-analyzer dist/stats.json",
     "purgecss": "node run-purgecss.js",
     "final-build": "npm run build && npm run purgecss",
-    "test": "ng test"
+    "test": "ng test",
+    "postinstall": "ngcc"
```

Comment:
why is this postinstall? I mean isn't ngcc short command already 🤔

Reply:
Also what's this command for exactly. m unaware of it

Reply:
It's to generate Ivy-compatible files for Angular. `postinstall` is actually called immediately after `npm install` is completed, so this isn't a command that a user runs.